sycl/include/CL/sycl/version.hpp.in

```diff
@@ -7,3 +7,6 @@
 //===----------------------------------------------------------------------===//
 
 #cmakedefine __SYCL_COMPILER_VERSION ${__SYCL_COMPILER_VERSION}
+#define __LIBSYCL_MAJOR_VERSION ${SYCL_MAJOR_VERSION}
```

Comment:
Where are SYCL_MAJOR_VERSION, SYCL_MINOR_VERSION, SYCL_PATCH_VERSION supposed to bedefined?  Why don't we just put hardcoded numbers in this file itself so that the way you change these numbers is by changing this source file?

Reply:
It seems the reason is because they should be aligned with the library soname which is configured  in the cmake files.

Reply:
Vlad is correct. These values come from `sycl/CMakeLists.txt`. They're updated once for all changes before release is tagged.